### PR TITLE
8.2.1.rc1 going to conda-forge/label/cdat_dev

### PR DESCRIPTION
**Pull request by muryanto1**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,31 +7,30 @@ package:
...
         - python
-        - cdms2 >=3.1.3
+        - cdms2
         - udunits2
         - {{ pin_compatible('numpy') }}
 
 test:
     commands:
-        -  export CDAT_ANONYMOUS_LOG=false && python -c "import genutil"
+        - python -c "import genutil"
```
Comment (jasonb5) [inline recipe/meta.yaml:34]: should keep this `export CDAT_ANONYMOUS_LOG=false`, cdms is import at somepoint.

**Review by jasonb5 — COMMENTED** [review]
Just the one change then LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (jasonb5) [reply recipe/meta.yaml:34]: nvm ignore this.

**Pull request merged**